atomic-design.md

```diff
@@ -28,7 +28,7 @@ An atomic element is any self contained code pattern that follows the Atomic Des
 
 ### Templates
 
-- To us: literally what atoms, molecules and organisms live on the wagtail templates
+- To us: literally which atoms, molecules and organisms live on the wagtail templates
```

Comment:
I wasn't getting into rewriting too much, just putting down what we had already with plans to make smaller changes from here on out. That said, I'm down w/ your change.